src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenOverridingAndHiding.vb

```diff
@@ -476,7 +478,11 @@ BC30308: 'Public Overrides Sub Format(i As Integer, j As Integer)' cannot overri
         End Sub
 
         <Fact()>
-        Public Sub OverloadingBasedOnOptionalParameters()
+        Public Sub OverloadingBasedOnOptionalParameters_A()
+            Dim useOpts = VisualBasicParseOptions.Default
+            If InternalSyntax.Parser.CheckFeatureAvailability(useOpts, InternalSyntax.Feature.ImplicitDefaultValueOnOptionalParameter) Then
```

Comment:
Note: Used the check if the feature is not enabled thus run the existing unit tests.